frontend/src/metabase/query_builder/components/EditableText/EditableText.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+
+import { Root, BorderedInput } from "./EditableText.styled";
+
+import { KEY_ESCAPE } from "metabase/lib/keyboard";
+
+interface Props {
+  value: string;
+  onChange?: (val: string) => void;
+}
+
+export const EditableText = ({ value, onChange }: Props) => {
```

Comment:
Should we export this as `default export` and also reexport it in `index.ts`? e.g. https://github.com/metabase/metabase/blob/8983b573e464d5c9d185455526d59457dde99ad7/frontend/src/metabase/core/components/Tab/index.ts#L1

@ranquild what's the convention for exporting TypeScript components?

Reply:
It's like on the line above

Reply:
Gotcha. I'll update it